subprojects/ide/ide.gradle.kts

```diff
@@ -82,3 +84,5 @@ classycle {
 testFilesCleanup {
     policy.set(WhenNotEmpty.REPORT)
 }
+
+makeIntegrationTestsDependOnSampleDir("subprojects/ide/src/main")
```

Comment:
Is there a way we can fail the build if the input is not declared?

Reply:
We can set a Test system property when declaring the inputs, then assert that system property during sample discovering process. Then a new test without declaring inputs fails immediately.

Reply:
It works immediately... https://builds.gradle.org/viewLog.html?buildId=34992820&tab=buildResultsDiv&buildTypeId=Gradle_Check_Quick_1_bucket22